dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -886,6 +886,7 @@ class StartupTest {
              '-v',
             '--profile',
             '--target=$target',
+            if (deviceOperatingSystem == DeviceOperatingSystem.ios) '--no-publish-port',
```

Comment:
Fixed `Mac_benchmark platform_view_macos__start_up` by limiting this to only be used for iOS.

Original PR failure: https://ci.chromium.org/ui/p/flutter/builders/prod/Mac_benchmark%20platform_view_macos__start_up/4413/overview

New PR passing: https://chromium-swarm.appspot.com/task?id=6557f6bd1e447a10